source/_integrations/unifiprotect.markdown

```diff
@@ -197,6 +204,15 @@ Use to dynamically set the message on a Doorbell LCD screen. This service should
 | `message`              | No       | The message you would like to display on the LCD screen of your Doorbell. Must be less than 30 characters.   |
 | `duration`             | Yes      | Number of minutes to display the message for before returning to the default message. The default is to not expire. |
 
+### Service unifiprotect.set_chime_paired_doorbells
+
+Use to set the paired doorbell(s) with a smart chime.
+
+| Service data attribute | Optional | Description                                                                                                  |
+| ---------------------- | -------- | ------------------------------------------------------------------------------------------------------------ |
+| `entity_id`            | No       | The smart chime you want to pair doorbells with (actually the "Play Chime" button).                          |
+| `doorbells`            | No       | A [target selector](https://www.home-assistant.io/docs/blueprint/selectors/#target-selector) for any number of doorbells you want to pair to the chime. |
```

Comment:
Please don't use full URLs; additionally: I'm also not sure how the target selector is relevant to mention or link in this case?